critiquebrainz/frontend/views/review.py

```diff
@@ -197,6 +197,7 @@ def create():
         flash(gettext("You can only write a review for an entity that exists on MusicBrainz!"), 'error')
         return redirect(url_for('search.selector', next=url_for('.create')))
     return render_template('review/write.html', form=form, entity_type=entity_type, entity=entity)
+    return render_template('review/entity/release_group.html' % review.entity_type, review=review, spotify_mappings=spotify_mappings, vote=vote)
```

Comment:
This part of code will never be reached. It's a good idea to test your changes beforehand. To connect to server with Spotify mappings you can update config file to have this (just make sure that you overwrite old variables):
```python
# Server with Spotify mappings
# https://github.com/metabrainz/mbspotify
MBSPOTIFY_BASE_URI = "http://mbspotify.musicbrainz.org/"
#MBSPOTIFY_ACCESS_KEY = ""
```